Link against libintl: these programs use gettext(3) but the Makefile.am
never adds LTLIBINTL (AM_GNU_GETTEXT([external])), which only works where
libintl lives in libc.
ld: error: undefined symbol: libintl_dgettext

Index: avahi-utils/Makefile.am
--- avahi-utils/Makefile.am.orig
+++ avahi-utils/Makefile.am
@@ -28,7 +28,7 @@ bin_PROGRAMS = avahi-browse avahi-resolve avahi-publis
 
 avahi_browse_SOURCES = avahi-browse.c sigint.c sigint.h
 avahi_browse_CFLAGS = $(AM_CFLAGS)
-avahi_browse_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la
+avahi_browse_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la $(LTLIBINTL)
 
 if HAVE_GDBM
 avahi_browse_SOURCES += stdb.h stdb.c
@@ -38,15 +38,15 @@ endif
 
 avahi_resolve_SOURCES = avahi-resolve.c sigint.c sigint.h
 avahi_resolve_CFLAGS = $(AM_CFLAGS)
-avahi_resolve_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la
+avahi_resolve_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la $(LTLIBINTL)
 
 avahi_publish_SOURCES = avahi-publish.c sigint.c sigint.h
 avahi_publish_CFLAGS = $(AM_CFLAGS)
-avahi_publish_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la
+avahi_publish_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la $(LTLIBINTL)
 
 avahi_set_host_name_SOURCES = avahi-set-host-name.c sigint.c sigint.h
 avahi_set_host_name_CFLAGS = $(AM_CFLAGS)
-avahi_set_host_name_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la
+avahi_set_host_name_LDADD = $(AM_LDADD) ../avahi-client/libavahi-client.la ../avahi-common/libavahi-common.la $(LTLIBINTL)
 
 install-exec-local:
 	$(MKDIR_P) $(DESTDIR)/$(bindir) && \
